rama-http/src/headers/x_robots_tag.rs

```diff
@@ -0,0 +1,58 @@
+use crate::headers::x_robots_tag_components::robots_tag_components::Parser;
+use crate::headers::x_robots_tag_components::RobotsTag;
+use crate::headers::Error;
+use headers::Header;
+use http::{HeaderName, HeaderValue};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct XRobotsTag(Vec<RobotsTag>);
+
+impl Header for XRobotsTag {
+    fn name() -> &'static HeaderName {
+        &crate::header::X_ROBOTS_TAG
+    }
+
+    fn decode<'i, I>(values: &mut I) -> Result<Self, Error>
+    where
+        Self: Sized,
+        I: Iterator<Item = &'i HeaderValue>,
+    {
+        let mut elements = Vec::new();
+        for value in values {
+            let mut parser = Parser::new(value.to_str().map_err(|_| Error::invalid())?);
```

Comment:
It's fine to swallow errors to turn them into other errors. However can you add at least an `inspect_error` and `tracing::debug` output them with a short description?:

Something like:

```
map_err(|err| {
     tracing::debug!(?err, "x-robots-tag header element decoding failure");
     Error::Invalid()
})?
```